qualification/report/generate_pdf.py

```diff
@@ -1009,11 +1036,13 @@ def _doc_outcome(section: Container, result: Result) -> None:
     if result.xfail_reason:
         section.append(NoEscape(r"\ "))
         section.append(f"({result.xfail_reason})")
-    section.append(Command("hspace", "1cm"))
+    section.append(Command("hspace", "0.5cm"))
```

Comment:
Any particular reason you changed it?

Reply:
Needed a bit more space to hold the overhead without the line wrapping